vendor/bat-native-ledger/src/bat/ledger/internal/promotion.cc

```diff
@@ -381,6 +387,10 @@ void Promotion::OnGetAllPromotions(
 
   if (list.size() > 0) {
     for (auto & item : list) {
+      if (item->legacyClaimed) {
+        ClaimTokens(std::move(item), [](const ledger::Result _){});
```

Comment:
should be `item->Clone` and not move, as if you move it then code after this point lost item pointer

Reply:
TY!

Reply:
Done